app/assets/stylesheets/logixRelease.css

```diff
@@ -129,4 +129,8 @@ a.btn.btn-outline-primary {
 
 a.btn.btn-outline-danger {
     margin-top: 10px;
-}
\ No newline at end of file
+}
+
+.card-shadow:hover {
+    box-shadow: 4px 4px #dfdfdf;
+}
```

Comment:
Add a smooth ease-in transition as well for this hover effect.

Reply:
![After2](https://user-images.githubusercontent.com/43378325/75040511-c99ae300-54e0-11ea-994e-a6994a81f792.gif)